about.html

```diff
@@ -0,0 +1,7 @@
+---
+title: About
+icon: info
+permalink: /about/
+---
+
+<meta http-equiv="Refresh" content=	"0; url=https://github.com/xenia-project/xenia/wiki/about">
```

Comment:
Even if you try to send the user to another URL this way, there's no guarantee they have this enabled. There should be content on the page noting the page move.

Reply:
Added a note about redirecting page.